docs/mkdocs.yml

```diff
@@ -8,6 +8,8 @@ nav:
   - User Guide:
     - 'Installing the CLI': 'user-guide/installing-cli.md'
     - 'Using the CLI': 'user-guide/using-cli.md'
+    - 'Command completion': 'user-guide/completion.md'
```

Comment:
Let's move these pages lower in the hierarchy so that there is a logical order to the TOC

Reply:
@pramod74 feel free to update this PR with the change; the assignment is for the PR review, so if you update the PR and mark it as 'ready for review' someone will pick it up again. Thanks!

Reply:
> Shall I move the 3 listed pages after 'Logging in to Microsoft 365' page?

I'd move `Command completion` after `Configuring the CLI` and `GitHub actions` before `Managing Microsoft 365 apps`

Reply:
Thanks, @waldekmastykarz and @appieschot.
I have pushed another commit after re-ordering pages in the User Guide. Can you kindly review?

Reply:
Have you also updated broken links?

Reply:
Done now, @waldekmastykarz. Kindly review. 
Thanks!

Reply:
Awesome! We'll review it shortly